packages/snippet/src/Domain/Repository/SnippetAreaRepositoryInterface.php

```diff
@@ -47,6 +47,16 @@ public function getOneBy(array $filters): SnippetAreaInterface;
      */
     public function findOneBy(array $filters): ?SnippetAreaInterface;
 
+    /**
+     * @param array{
+     *     uuid?: string,
+     *     uuids?: string[],
+     *     webspaceKey?: string,
+     *     areaKey?: string,
+     * } $filters
+     */
+    public function findSnippetUuidBy(array $filters): ?string;
```

Comment:
avoid the `Snippet` prefix we are already in the Snippet Repository.


```suggestion
    public function findOneUuidBy(array $filters): ?string;
```